modular_skyrat/modules/colony_fabricator/code/design_datums/appliances.dm

```diff
@@ -0,0 +1,38 @@
+// Machine categories
+
+#define FABRICATOR_CATEGORY_APPLIANCES "/Appliances"
+#define FABRICATOR_SUBCATEGORY_POWER "/Power"
+
+// Techweb node that shouldnt show up anywhere ever specifically for the fabricator to work with
+
+/datum/techweb_node/colony_fabricator_appliances
+	id = "colony_fabricator_applicanes"
+	display_name = "Colony Fabricator Appliance Designs"
+	description = "Contains all of the colony fabricator's appliance machine designs."
+	design_ids = list(
+		"wall_multi_cell_rack",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000000000000000) // God save you
+	hidden = TRUE
+	show_on_wiki = FALSE
+	starting_node = TRUE
+
+// Lets the colony lathe make more colony lathes but at very hihg cost, for fun
+
+/datum/design/wall_mounted_multi_charger
+	name = "Mounted Multi-Cell Charging Rack"
```

Comment:
~~Is this comment meant to be related to the mounted multi-cell charger? Or is the Colony Fab design missing?~~
Comment copypaste augh